state/execution.go

```diff
@@ -23,6 +24,8 @@ import (
 // It exposes ApplyBlock(), which validates & executes the block, updates state w/ ABCI responses,
 // then commits and updates the mempool atomically, then saves state.
 
+var config = cfg.DefaultBaseConfig()
```

Comment:
is this the real config used in production environment? from the name, I thought it's only a default config?